tutorials/CMakeLists.txt

```diff
@@ -505,6 +505,15 @@ if(ROOT_pyroot_FOUND)
              sql/sqlfilldb.py       # same as the C++ case
              sql/sqlselect.py       # same as the C++ case
              )
+
+  if(NOT pyroot_experimental)
+    list(APPEND pyveto roofit/rf501_simultaneouspdf.py)
```

Comment:
Correction: this tutorial should not be vetoed, it does not use the `Import` pythonization.